From bf53d4d25fa3e76005771243c8541a0a8950e0c7 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Thu, 22 Dec 2016 18:13:42 +0800
Subject: [PATCH 1402/5242] MLK-13631-02: ARM: dts: Change i.MX6SL CSI module
 compatible string

commit  af11d0d9d1eff5fc6973d8fb04efe084c97ccf3a from
https://source.codeaurora.org/external/imx/linux-imx.git

Change i.MX6SL CSI module compatible string from "fsl,imx6s-csi"
to "fsl,imx6sl-csi".

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sl.dtsi b/arch/arm/boot/dts/imx6sl.dtsi
index 422913b..2c057f9 100644
--- a/arch/arm/boot/dts/imx6sl.dtsi
+++ b/arch/arm/boot/dts/imx6sl.dtsi
@@ -768,7 +768,7 @@
 			};
 
 			csi: csi@20e4000 {
-				compatible = "fsl,imx6s-csi";
+				compatible = "fsl,imx6sl-csi";
 				reg = <0x020e4000 0x4000>;
 				interrupts = <0 7 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SL_CLK_DUMMY>,
-- 
1.7.9.5

